### PR TITLE
Fix issue 15838 	Many Win32 API callback functions miss extern(Windows)

### PR DESCRIPTION
**Pull request by qchikara**
Add extern (Windows) to aliases and struct fields.

### CODE REVIEW

File: src/core/sys/windows/commctrl.d [hunk 1]
```diff
@@ -3869,7 +3869,7 @@ static if (_WIN32_WINNT >= 0x501) {
     }
     alias LVINSERTGROUPSORTED* PLVINSERTGROUPSORTED;
 
-    alias int function(INT, INT, VOID*) PFNLVGROUPCOMPARE;
+    extern (Windows) alias int function(INT, INT, VOID*) PFNLVGROUPCOMPARE;
```
Comment (DmitryOlshansky) [inline src/core/sys/windows/commctrl.d]: Shouldn't it be `alias PFNLVGROUPCOMPARE = extern(Windows) int function(INT, INT, VOID*);` ?

Comment (qchikara) [inline src/core/sys/windows/commctrl.d]: This is OK,
and extern(Windows){  aliases ...  }, too.

Comment (qchikara) [inline src/core/sys/windows/commctrl.d]: Ah, old style aliases should be fixed ...?

Comment (DmitryOlshansky) [inline src/core/sys/windows/commctrl.d]: Yes, that'd be great as a separate commit.

Comment (qchikara) [inline src/core/sys/windows/commctrl.d]: I think the change of styles should be made at all.
So, these core.sys.windows.* modules have thousands of aliases, mostly in old styles.
I misses such a skill to do efficiently :P

Comment (DmitryOlshansky) [inline src/core/sys/windows/commctrl.d]: Okay let's ignore them for now.

**Comment by DmitryOlshansky** [comment]
LGTM

**Comment by DmitryOlshansky** [comment]
Auto-merge toggled on
